### PR TITLE
fix: login page readable error on ak server error response

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Continuing this ticket: [UI-901](https://linear.app/autokitteh/issue/UI-901/include-discord-connection-error-message)

Log the proper error from the backend - log only to the console, not the system logger.

## Linear Ticket
https://linear.app/autokitteh/issue/UI-903/fix-login-error-handling

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [x] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/templates/descopeMiddleware.tsx [hunk 1]
```diff
@@ -57,16 +57,30 @@ export const DescopeMiddleware = ({ children }: { children: ReactNode }) => {
 				const error = await getLoggedInUser();
 				if (error) {
 					addToast({
-						message: t("errors.loginFailed", { error }),
+						message: t("errors.loginFailedExtended", { error }),
```
Comment (J1za) [inline src/components/templates/descopeMiddleware.tsx:60]: Why message Extended inside toast?

Comment (RonenMars) [reply src/components/templates/descopeMiddleware.tsx:60]: done

Comment (RonenMars) [reply src/components/templates/descopeMiddleware.tsx:60]: also removed {error} from the translation function.

**Review by J1za — APPROVED** [review]

File: src/components/templates/descopeMiddleware.tsx (hunk 1 above)
Comment (J1za) [reply src/components/templates/descopeMiddleware.tsx]: Thank you

**Pull request merged**